standalone/src/recording/components/recording_view/index.tsx

```diff
@@ -193,7 +195,7 @@ const RecordingView = () => {
                     url={pictures[speakingProfile.id]}
                 />
                 <span style={{marginLeft: '8px'}}>{getUserDisplayName(speakingProfile)}</span>
-                <span style={{fontWeight: 400}}>{' is talking...'}</span>
+                <span style={{fontWeight: 400}}>{untranslatable(' ')}{formatMessage({defaultMessage: 'is talking...'})}</span>
```

Comment:
Is that best practices for what we should be doing? Cool, I never knew. We always just put a blank space or a blank space code. :)

Reply:
Probably not but I really wanted to have the linter on and complaining about missing translations going forward and to do that I had to find a way to mute errors from strings that are not meant to be translated. The [eslint plugin](https://formatjs.io/docs/tooling/linter/) supports exclude rules but they rely on DOM element tags and I couldn't find an elegant way to apply that so a dedicated function seemed like a decent option. Happy to iterate further on this though if we figure out a better solution.